include/picongpu/fields/FieldManipulator.hpp

```diff
@@ -83,7 +83,7 @@ class FieldManipulator
                 }
 
                 /* if sliding window is active we disable absorber on bottom side*/
-                if (MovingWindow::getInstance().isSlidingWindowActive() && i == BOTTOM) continue;
+                if (MovingWindow::getInstance().isSlidingWindowActive(currentStep) && i == BOTTOM) continue;
```

Comment:
@psychocoderHPC why did you change this as well?

Reply:
forget about it, I saw it below... makes sense

Reply:
Because without the time step it is not possible to check if the moving window is stopped.
As soon as the window stops the bottom absorber is enabled else the laser get reflected

Reply:
Yes, overlooked this